test/Unit/Utility/TypeExtensionsTests.cs

```diff
@@ -372,8 +372,8 @@ public void TestTypedDataToObjectJson6()
             Assert.Equal(2, actualArray.Length);
             Assert.Equal("Foo", actualArray[0]);
 
-            Assert.IsType<Hashtable>(actualArray[1]);
-            var nestedHash = (Hashtable)actualArray[1];
+            Assert.IsType<OrderedHashtable>(actualArray[1]);
```

Comment:
Is there a reason why we expect this type to change from `Hashtable` to `OrderedHashtable`? Did the protobuf get updated?

Reply:
In `PowerShell 7.4` `ConvertFrom-Json` returns an ordered hashtable instead of a hashtable (for more information, please see https://github.com/PowerShell/PowerShell/pull/17405). I have updated the PR description to include this information.

Reply:
@Francisco-Gamino I'm wondering if we need to update this code as well: https://github.com/Azure/azure-functions-powershell-worker/blob/1fcb262293094e418cd1c216ce35f4ba193971fb/src/Utility/TypeExtensions.cs#L125

I think we need to do the same to `OrderedHashtable` as well. If we don't, hashtables received as function parameters will be case-sensitive.

Reply:
> @Francisco-Gamino I'm wondering if we need to update this code as well:
> 
> https://github.com/Azure/azure-functions-powershell-worker/blob/1fcb262293094e418cd1c216ce35f4ba193971fb/src/Utility/TypeExtensions.cs#L125
> 
> I think we need to do the same to `OrderedHashtable` as well. If we don't, hashtables received as function parameters will be case-sensitive.

@anatolib -- As per our conversation, I've removed the logic to create case-insensitive hashtable from the ConvertFromJson output. This will be a breaking change in the `PowerShell 7.4` language worker. 

Also, I've created an issue to create a ConvertFromJson cmdlet to support case insensitive keys. Fore more information, please see https://github.com/Azure/azure-functions-powershell-worker/issues/909